File: LAPORAN PRATIKUM/MODUL7/CODEBLOKS DAN PYTHON/SOAL 1/PRAK601-2210817210014-MuhammadQoharyRawidafany.py
bar, kol = map(int, input().split())
angka = []
x = 0
isi = list(map(int, input().split()))
for i in range (bar):
    angka.append(isi[x : x + kol])
    x = x + kol
for bar in angka:
    for isi in bar:
        print(isi, end=' ')
    print()

File: LAPORAN PRATIKUM/MODUL7/CODEBLOKS DAN PYTHON/SOAL 1/PRAK601-2210817210014-MuhammadQoharyRawidafany.c
#include <stdio.h>
int main ()
{
    int bil[10][10],bar,kol;
    scanf("%d %d",&bar,&kol);
    for (int a=0;a<bar;a++){
        for (int b=0;b<kol;b++){
        scanf("%d",&bil[a][b]);
        }
    }
    printf("");
    for (int a=0;a<bar;a++){
        for(int b=0;b<kol;b++){
        printf(" %d ",bil[a][b]);
        }
        printf("\n");
    }
}
